monkey/monkey_island/cc/app.py

```diff
@@ -109,6 +108,11 @@ class CustomConfirmRegisterForm(ConfirmRegisterForm):
             "Email", default="dummy@dummy.com", validators=[validate_no_user_exists_already]
         )
 
+        def to_dict(self, only_user):
+            dict = super().to_dict(only_user)
+            dict.update({'roles': ['ISLAND']})
```

Comment:
We can build `Roles` object here and pass a description but I'm not sure where those descriptions are/would be used. Could you explain?

Reply:
If we have Role objects here maybe `flask-security-too` will create those roles and we don't need an explicit call to `_create_roles(user_datastore)`?

Reply:
The datastore has specific functions to do that. Why would we want to create `Role` object when we define user_datastore which its functionality is to do that and more?

Reply:
The descriptions probably are not even needed at this point. Maybe later?

Reply:
The datastore is used by `flask-security-too`, we don't need to explicitly use it where we don't have to. If we have no users with a specific role we also don't really need that role in the database. It would be nice if `flask-security-too` could create the roles for us

Reply:
The only way to create the roles is by using the `db` or using the `UserDatastore`.